packages/mnid/LICENSE

```diff
@@ -0,0 +1,98 @@
+The MIT License (MIT)
+
+Copyright (C) 2018 BiiLabs, Co. Ltd. and Contributors
```

Comment:
Should be `2018-2019`

Reply:
The formal name is `BiiLabs Co., Ltd.` Be aware of the comma.

Reply:
I notice that this already fixed in the `develop` branch, so I rebased this branch.